src/docker.test.ts

```diff
@@ -90,42 +131,52 @@ describe("Docker images", (): void => {
     expect(docker.CACHE_HIT).toBe("cache-hit");
   });
 
+  test("exports DOCKER_IMAGES_LIST", (): void => {
+    expect(docker.DOCKER_IMAGES_LIST).toBe("docker-images-list");
+  });
+
   test("exports DOCKER_IMAGES_PATH", (): void => {
     expect(docker.DOCKER_IMAGES_PATH).toBe("~/.docker-images.tar");
   });
 
   test("are loaded on cache hit", async (): Promise<void> => {
     await mockedLoadDockerImages(true);
 
-    expect(util.execBashCommand).lastCalledWith(
-      `docker load --input ${docker.DOCKER_IMAGES_PATH}`
-    );
-
     /* The cache must be restored before the Docker images can be loaded. This
      * at least checks that the calls are made in the right order, but doesn't
      * ensure that the cache finished restoring before the Docker images started
      * loading.
      */
+    expect(core.saveState).toHaveBeenCalledTimes(1);
+    expect(util.execBashCommand).toHaveBeenCalledTimes(1);
```

Comment:
These two lines belong above the comment, which pertains to `assertCalledInOrder`. I would keep the blank line as well to avoid confusing Python developers, who may be accustomed to comments following rather than preceding what they describe.